src/base.jl

```diff
@@ -91,20 +91,25 @@ struct System <: IS.InfrastructureSystemsType
     end
 end
 
-function System(data, base_power, internal; kwargs...)
+function System(data, base_power::Float64, internal; kwargs...)
```

Comment:
Should we make this `base_power::Number` so that the user can use `System(100)`?

Reply:
ok